src/SFML/Audio/OutputSoundFile.cpp

```diff
@@ -36,14 +36,6 @@ namespace sf
 OutputSoundFile::OutputSoundFile() = default;
 
 
-////////////////////////////////////////////////////////////
-OutputSoundFile::~OutputSoundFile()
-{
-    // Close the file in case it was open
-    close();
-}
```

Comment:
Why is it okay to not call `close()` at destruction?

Reply:
`close()` only deletes an `std::unique_ptr`'s managed object, which is what a default constructor would do anyways.

Reply:
Kimci is right. The [corresponding commit](https://github.com/SFML/SFML/pull/2329/commits/cacba32491024bb9bbc433e4c4db56303f642b29) described my rationale.